components/var_dumper/introduction.rst

```diff
@@ -62,6 +62,17 @@ current PHP SAPI:
     You'll also learn how to change the format or redirect the output to
     wherever you want.
 
+.. tip::
+
+    In order to have the ``dump()`` function always available, you can install
+    it globally in your dev environment:
+
+    #. Run ``composer global require symfony/var-dumper``;
+    #. Add ``auto_prepend_file = ${HOME}/.composer/vendor/autoload.php``
+       to your ``php.ini`` file;
+    #. (Don't forget to run ``composer global update`` from time to time to always
```

Comment:
I don't think "update this component" adds much extra value.

I've added the parenthesis because it's not really a next step in the installation process, it's just something you have to do in the time after it. Another option is to add it as a normal paragraph after the list